elastic/kibana
# [ResponseOps][Flapping] Update flapping code once the flapping lookback value is configurable

Resolves https://github.com/elastic/kibana/issues/145929

## Summary

Updates previous flapping tests to use the new flapping settings configs.
Updates flapping logic to use flapping configs instead of hardcoded values. Calls the flapping api on every rule execution, and then passes in the flapping settings to the rule executors so they can be used by the rule registry.


### Checklist



### To verify

I think it's helpful to hide the whitespace when reviewing this pr.

- The flapping logic should remain the same, and all previous tests should pass. I only updated them to pass in the flapping settings.
- Create rules, and set flapping settings in the ui and see the flapping behavior change for your rules. 
- Verify that the `x-pack/test/alerting_api_integration/spaces_only/tests/alerting/event_log.ts` run with the new flapping configs and output results we would expect
